functions/webhooks/taskrouterCallback.protected.ts

```diff
@@ -53,14 +53,24 @@ const wait = (ms: number): Promise<void> => {
   });
 };
 
-const isCreateContactTask = (eventType: EventType, taskAttributes: { isContactlessTask?: boolean }) => 
-  eventType === TASK_CREATED_EVENT && taskAttributes.isContactlessTask;
+const isCreateContactTask = (
+  eventType: EventType,
+  taskAttributes: { isContactlessTask?: boolean },
+) => eventType === TASK_CREATED_EVENT && taskAttributes.isContactlessTask;
```

Comment:
Be aware that this was updated, you will need to fix the merge conflict, and please keep the version from `master`.

Reply:
@GPaoloni Alright.